pkgs/development/tools/continuous-integration/hercules-ci-agent/default.nix

```diff
@@ -15,17 +28,47 @@ let
           makeWrapper $out/libexec/hercules-ci-agent $out/bin/hercules-ci-agent --prefix PATH : ${lib.escapeShellArg (makeBinPath bundledBins)}
         '';
       })
-      (addBuildTools [ makeWrapper ] (justStaticExecutables haskellPackages.hercules-ci-agent));
-in pkg.overrideAttrs (o: {
+      (addBuildTools [ makeBinaryWrapper ] (justStaticExecutables haskellPackages.hercules-ci-agent));
+in pkg.overrideAttrs (finalAttrs: o: {
     meta = o.meta // {
       position = toString ./default.nix + ":1";
     };
     passthru = o.passthru // {
+
+      tests.help = runCommand "test-hercules-ci-agent-help" { } ''
+        (${finalAttrs.finalPackage}/bin/hercules-ci-agent --help 2>&1 || true) | grep -F -- '--config'
+        (${lib.getExe finalAttrs.finalPackage} --help 2>&1 || true) | grep -F -- '--config'
+        touch $out
+      '';
```

Comment:
This help test doesn't seem to run on Linux, is it because the update isn't recursive?

Reply:
Ok I fixed the update.